hw3/01-characters.js

```diff
@@ -0,0 +1,33 @@
+const character = document.getElementsByClassName("character");
+
+async function getCharacters() {
+  const url = "https://thronesapi.com/api/v2/Characters";
+  try {
+    let response = await fetch(url);
+    return await response.json();
+  } catch (erorr) {
+    console.log(error);
+  }
```

Comment:
console.error is useful for developers and debugging. However, a casual user is likely not going to check the console dev tools.

It would be more user friendly to also display/append an error message on the web page.